src/uct/ib/dc/accel/dc_mlx5.c

```diff
@@ -800,8 +800,8 @@ static UCS_CLASS_INIT_FUNC(uct_dc_mlx5_iface_t, uct_md_h md, uct_worker_h worker
                              sizeof(struct mlx5_wqe_data_seg));
 
     /* TODO: only register progress when we have a connection */
-    uct_worker_progress_add_safe(worker, uct_dc_mlx5_iface_progress, self,
-                                 &self->super.super.super.super.prog);
+    uct_dc_iface_progress_enable(&self->super.super.super.super.super,
```

Comment:
Is it possible to enable/disable progress from base DC iface constructor to avoid calling it for both verbs and PRM?